src/pages/Chat/index.jsx

```diff
@@ -82,7 +80,6 @@ const Chat = () => {
         {Messages}
         {loading && (
           <MessageRow
-            // @ts-ignore
```

Comment:
Please left ts-ignore flag, since without it I'm receiving JSDoc error

![imagen](https://user-images.githubusercontent.com/66505715/156258376-cc02d4c2-14e2-4371-9c49-3a93c9e84cb8.png)

Reply:
not having any reference on this, I decided to remove it